makefile

```diff
@@ -179,6 +179,13 @@ $(buildDir)/go-test-config:cmd/go-test-config/make-config.go
 	GOPATH=$(shell dirname $(shell pwd)) $(gobin) build -o $@ $<
 #end generated config
 
+# parse a host.create file and set expansions
+parse-host-file:$(buildDir)/parse-host-file
```

Comment:
"expansions.yml":$(buildDir)/parse-host-file

with an alias target if you want.

Reply:
I don't see this change.

Reply:
It's on line 187